TIPs/tip-800-SGX.md

```diff
@@ -36,19 +36,25 @@ Why it's necessary for Ternoa?
 
 ### External Interfaces
 
-onchain interfaces:
-
-```rust
-interface { 
-  
-}
-```
 
 ### Existing Interfaces changed
 
 ```rust
 interface {
-  
+
+  register_enclave(origin: OriginFor<T>, api_uri: TextFormat)
```

Comment:
How to remove/unregister an enclave from blockchain totally? Because Assign/Unassign are cluster-wise operations.